src/utils.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable import/prefer-default-export */
+const todos = JSON.parse(localStorage.getItem('todos'))
+  ? JSON.parse(localStorage.getItem('todos'))
+  : [];
```

Comment:
- [OPTIONAL] Let's simplify this logic and economies lines of code using the OR(||) operator.
```suggestion
const todos = JSON.parse(localStorage.getItem('todos')) || [];
```